modules/solid_mechanics/include/materials/TorchPlasticityStressUpdate.h

```diff
@@ -0,0 +1,78 @@
+//* This file is part of the MOOSE framework
```

Comment:
As we discussed, this model and the related files should either go into solid_mechanics/test/ or into the repository for the SciDAC. I'd say the SciDAC repository is probably the better place at the moment. Maybe we can make a simple model based on this that is actually physical that could go in the solid_mechanics module.